xl/trax/track.py

```diff
@@ -891,11 +891,21 @@ def get_rating(self):
         :rtype: int
         """
         try:
-            rating = float(self.get_tag_raw('__rating'))
-        except (TypeError, KeyError, ValueError):
+            f = self._get_format_obj()
+            rating = None
+            if f and '__rating' in f.tag_mapping and settings.get_option('collection/write_rating_to_audio_file_metadata', False):
+                rating = self.get_tag_disk('__rating')
+            if rating:
+                rating = rating[0]
+            elif rating is None:
+                rating = self.get_tag_raw('__rating')
```

Comment:
The `if rating` / `elif rating is None` clauses do not handle the case where `rating` is falsey but not `None`. This doesn't appear to be a logical possibility, so why not reformulate this so that it is inherently inclusive with either `if rating` / `else` or `if rating is None` / `else`?